src/Actions/Action.php

```diff
@@ -34,6 +34,9 @@ public function getClassForJs()
     public function getTitle()
     {
         $title = $this->title ?? niceTitle(collect(explode('\\', get_class($this)))->last());
+        if (strpos(__('admin.'.$title), 'admin.') === false) {
```

Comment:
@aleixgil està bé, però enlloc d'admin, crec que s'hauria d'afegir un paràmetre al config/thrust.php que es digués translations prefix, i aqui fer-lo anar. Depèn del projecte, pot no dir-se admin.

Reply:
bueno, jo crec que no cal ni el parametre ni el admin
si es passa el title, ja es passara traduït, per tant if title return title

Reply:
@PauRevo  ara que ho veig, ja estava fet lo del prefix a la config de thrust. Només l'hauria de pillar hehe
@BadChoice no es tradueix sol...

Reply:
@BadChoice el titol, hi és tot sovint (o sempre). L'aleix mira que si se li passa un títol sense prefix, li afegeix i el tradueix.
L'altre opció és passa el titol traduit sempre, si.

Reply:
Bueno, he trobat llocs on no hi havia $titol, i pot ser que no interessi posar-ho, pel que sigui. Per això primer miro si hi ha $titol, i si hi ha miro si existeix traducció, sinó mostro $titol tal cual (que seria una altre opció, en cas que no es vulgui fer la traducció, o no calgui pq es la mateixa paraule en tots els idiomes, i en cas que no es vulgui deixar el nom de la classe com a titol per defecte).

Reply:
si, la idea es passar el titol traduït des de l'app que toqui
Ja que l'action la crearas al projecte, per tant no costa res posar en ell matei $this->title = __('xxx.yyy') i que no depengui de thrust, si no es posa titol, llavors que faci el nicetitle i avall

Reply:
<img width="1396" alt="Captura de pantalla 2021-04-19 a las 13 06 04" src="https://user-images.githubusercontent.com/29859922/115226885-04fa6700-a110-11eb-9f4a-a6f5c76f91a3.png">

És el que havia provat al principi, però no li agrada.

Reply:
L'altre manera es al projecte sobrescriure el mètode i pasar la traducció, que és com estava fet a alguns llocs. Però tenint el $title, penso que s'hauria de aprofitar per fer la traducció.

Reply:
Ja ho he actualitzat amb lo que deiem al principi, utilitzant un paràmetre que ja hi havia a la config de thrust, que és el prefix de traducció

Reply:
@BadChoice Treu versió si ho veus bé!!